### ISSUE TITLE
Check if webcrypto is available at all

### ISSUE THREAD

**Issue by Lex-2008** [Lex-2008/pkc#2]
We should detect availability of webcrypto and show a nice message if it's not available. And not allow user to do anything else.

**Comment by rileythomp** [comment]
Does this just need a check for ```(window.crypto.subtle != null && window.crypto.subtle != undefined)``` ?

**Comment by Lex-2008** [comment]
Yes. Javascript-wise I would do just `if(!window.crypto.subtle)` for simplicity, and if it's the case - disable all controls on the page and show an error message in the error message box.
